entrypoint.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-
 set -o errexit
 
+# init /srv if empty
+DIST_FILE=/srv/pmm-distribution
+if [ ! -f $DIST_FILE ]; then
+    echo "File $DIST_FILE doesn't exist. Initizlize /srv..."
+    echo docker > $DIST_FILE
+    mkdir -p /srv/{clickhouse,grafana,logs,postgres,prometheus,nginx,victoriametrics}
```

Comment:
I think, we are migrating `postgres` to contain the version, i.e. `postgres14` if I look at this https://github.com/percona/pmm-server/pull/362/files#diff-500a0e8dcb2879d903640ba46031362cdc923d030e278cf2a681c24765cc87b0R41

Reply:
OK if this will be merged prior to that migration PR.